packages/app/pages/Api/organization.tsx

```diff
@@ -0,0 +1,113 @@
+/**
+ * This exports a React element which displays a list of all organizations,
+ * and provides a section to select a single organization out of that list
+ *
+ * @package: REA app
+ * @author:  Steven Fontaine <fontainesw@msoe.edu>
+ * @since:   2018-02-08
+ */
+
+import * as React from "react";
+import "./api.css";
+import getAllOrganizations from "../../../ui-bindings/Organization/getAllOrganizations";
+import { concatArray } from "./agent";
+import getOrganizationById from "../../../ui-bindings/Organization/getOrganizationById";
+
+export const Organization = (props) => {
+  let organization = props.organization;
+  return(
+    <div>
+      <div>id: {organization.id}</div>
+      <div>name: {organization.name}</div>
+      <div>type: {organization.type}</div>
+      <div>image: {organization.image}</div>
+      <div>note: {organization.note}</div>
+      <div>ownedEconomicResources: {concatArray(organization.ownedEconomicResources)}</div>
+      <div>agentProcesses: {concatArray(organization.agentProcesses)}</div>
+      <div>agentPlans: {concatArray(organization.agentPlans)}</div>
+      <div>agentEconomicEvents: {concatArray(organization.agentEconomicEvents)}</div>
+      <div>agentCommitments: {concatArray(organization.agentCommitments)}</div>
+      <div>agentRelationships: {concatArray(organization.agentRelationships)}</div>
+      <div>agentRoles: {concatArray(organization.agentRoles)}</div>
+      <div>agentRecipess: {concatArray(organization.agentRecipes)}</div>
+      <br/>
+    </div>
+  );
+};
+
+const OrganizationField = (props) => {
+  return(
+    <div>
```

Comment:
[Duplicate of UP-94] I would remove the ternary operator from here. This isn't what the ternary operator was made to do, and I think it leads to unnecessary code complexity. Consider an if-elseif-else block instead

Reply:
there is no ternary operator here.